tests/unit/main/services/ai/secondBrain/secondBrainRefresh.service.test.ts

```
@@ -0,0 +1,414 @@
+    expect(
+      (await secondBrain.readStateFile())?.sourceCursors.sessions,
+    ).toBeUndefined();
+    await expect(
+      secondBrain.readPage(durableOperation.pageId),
+    ).rejects.toMatchObject({ code: 'NOT_FOUND' });
+  });
+
+  it('accepts OKF documents with type frontmatter and no title', async () => {
+    const operationWithoutTitle = {
+      ...durableOperation,
+      content: durableOperation.content.replace('title: Revenue validation\n', ''),
```

ESLint

[error] Replace `'title:·Revenue·validation\n',·''` with `⏎········'title:·Revenue·validation\n',⏎········'',⏎······`

(prettier/prettier)